_includes/head.html

```diff
@@ -17,6 +17,7 @@
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
     <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
     <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
+    <link rel="shortcut icon" type="image/png" href="../img/logos/logo_icon_dark.png">
```

Comment:
Instead of a relative link, this should use one of the Jekyll variables for the "top" of the website.